pkg/cmd/util/output/backup_describer.go

```diff
@@ -253,15 +258,34 @@ func DescribeBackupStatus(d *Describer, backup *velerov1api.Backup, details bool
 
 		d.Printf("Persistent Volumes:\n")
 		for _, snap := range snapshots {
-			printSnapshot(d, snap.Spec.PersistentVolumeName, snap.Status.ProviderSnapshotID, snap.Spec.VolumeType, snap.Spec.VolumeAZ, snap.Spec.VolumeIOPS)
+			describeSnapshot(d, snap.Spec.PersistentVolumeName, snap.Status.ProviderSnapshotID, snap.Spec.VolumeType, snap.Spec.VolumeAZ, snap.Spec.VolumeIOPS)
 		}
 		return
 	}
 
 	d.Printf("Persistent Volumes: <none included>\n")
 }
 
-func printSnapshot(d *Describer, pvName, snapshotID, volumeType, volumeAZ string, iops *int64) {
+func describeBackupResourceList(d *Describer, backup *velerov1api.Backup, veleroClient clientset.Interface) {
+	buf := new(bytes.Buffer)
+	if err := downloadrequest.Stream(veleroClient.VeleroV1(), backup.Namespace, backup.Name, velerov1api.DownloadTargetKindBackupResourceList, buf, downloadRequestTimeout); err != nil {
+		d.Printf("Resource List:\t<error getting backup resource list: %v>\n", err)
```

Comment:
What does this error look like if you run `velero backup describe --details` for a backup that doesn't have the resource list (e.g. from v1.0)? Wondering if there's a way to discern when the list doesn't exist and show a more friendly error.

Reply:
Unfortunately Stream doesn't return the status code and the error message will be different depending on the storage provider used. E.g. for Azure:

```
Resource List:  <error getting backup resource list: request failed: <?xml version="1.0" encoding="utf-8"?><Error><Code>BlobNotFound</Code><Message>The specified blob does not exist.
RequestId:4628a164-001e-0136-645d-496ab8000000
Time:2019-08-02T18:11:07.1078302Z</Message></Error>>
```

I agree it's not the best UX for older backups, how about modifying `Stream` to return a more user-friendly error when the status code is 404, rather than showing the raw error?

Reply:
yeah, i think that makes sense

Reply:
I think we're just waiting on addressing this comment before merge.